inspect_builder.go

```diff
@@ -24,8 +24,8 @@ type BuilderInfo struct {
 	CreatedBy       builder.CreatorMetadata
 }
 
-func (c *Client) InspectBuilder(name string, daemon bool) (*BuilderInfo, error) {
-	img, err := c.imageFetcher.Fetch(context.Background(), name, daemon, false)
+func (c *Client) InspectBuilder(ctx context.Context, name string, daemon bool) (*BuilderInfo, error) {
+	img, err := c.imageFetcher.Fetch(ctx, name, daemon, false)
```

Comment:
This change (and other builder inspection changes) are basically unrelated to the feature, but I noticed they could use some refactoring was I was working. Happy to pull into their own PR.